### PR TITLE
DOCS-8441 LLM Observability Documentation Navigation Update

### PR DESCRIPTION
**Pull request by alai97**
Move pages around in the LLM Observability section to fit a new doc structure. Also documents the LLM Observability permission in the Setup page for users + adds evaluation definitions in the Concepts and Terms page.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/llm_observability/terms/_index.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+When used alongside embedding spans, retrieval spans can provide visibility into retrieval augmented generation (RAG) operations.
+
+Retrieval spans typically do not have child spans, as they represent a standalone retrieval step.
+
+## Traces
+
+A trace represents the work involved in processing a request in your LLM application, and consists of one or more nested spans. A *root span* is the first span in a trace, and marks the beginning and end of the trace.
+
+Datadog's LLM Observability product is designed to support observability for LLM applications with varying complexity. Based on the structure and complexity of your traces, you can unlock the following features of LLM Observability:
```
Comment (alai97) [inline content/en/llm_observability/terms/_index.md:115]:
```suggestion
LLM Observability supports observability for LLM applications with varying complexity. Based on the structure and complexity of your traces, you can access the following features of LLM Observability:
```

File: content/en/llm_observability/terms/_index.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+LLM inference traces are composed of a single LLM span.
+
+{{< img src="llm_observability/llm-observability-llm-span.png" alt="A single LLM span" style="width:100%;" >}}
+
+Tracing individual LLM inferences unlocks basic LLM Observability features, allowing you to:
+
+1. Track inputs and outputs to your LLM calls
+2. Track token usage, error rates, and latencies for your LLM calls
+3. Break down important metrics by model and model provider
```
Comment (alai97) [inline content/en/llm_observability/terms/_index.md:127]:
```suggestion
1. Track inputs and outputs to your LLM calls.
2. Track token usage, error rates, and latencies for your LLM calls.
3. Break down important metrics by model and model provider.
```

**Review by davit-bar — APPROVED** [review]

**Comment by neko-dd** [comment]
Created DOCS-8627 for docs review :)

File: content/en/llm_observability/quickstart.md [hunk 3]
```diff
@@ -1,90 +1,91 @@
...
 
-### 4. View the trace
+1. View the trace of your LLM call on the **Traces** tab [of the **LLM Observability** page][3] in Datadog.
 
-A trace of your LLM call should appear in [the Traces tab][3] of LLM Observability in Datadog.
-
-{{< img src="llm_observability/quickstart-trace.png" alt="An LLM Observability trace displaying a single LLM request" style="width:100%;" >}}
+   {{< img src="llm_observability/quickstart-trace.png" alt="An LLM Observability trace displaying a single LLM request" style="width:100%;" >}}
 
 The trace you see is composed of a single LLM span. The `ddtrace-run` command automatically traces your LLM calls from [Datadog's list of supported integrations][10].
```
Comment (lievan) [inline content/en/llm_observability/quickstart.md:73]: Clicking this link 404's, i think the autoinstrumentation path was changed?

File: content/en/llm_observability/terms/_index.md [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+
+{{< img src="llm_observability/llm-observability-llm-span.png" alt="A single LLM span" style="width:100%;" >}}
+
+Tracing individual LLM inferences unlocks basic LLM Observability features, allowing you to:
+
+1. Track inputs and outputs to your LLM calls.
+2. Track token usage, error rates, and latencies for your LLM calls.
+3. Break down important metrics by model and model provider.
+
+The SDK provides integrations to automatically capture LLM calls to specific providers. See [Auto-instrumentation][3] for more information. If you are using an LLM provider that is not supported, you must [manually instrument your application][4].
```
Comment (alai97) [inline content/en/llm_observability/terms/_index.md:127]:
```suggestion

For a detailed example, see the [LLM Monitoring Jupyter notebook][7] which demonstrates how to create and trace an LLM call.

The SDK provides integrations to automatically capture LLM calls to specific providers. See [Auto-instrumentation][3] for more information. If you are using an LLM provider that is not supported, you must [manually instrument your application][4].
```

File: content/en/llm_observability/terms/_index.md [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+## Further Reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: https://react-lm.github.io/
+[2]: /llm_observability/setup/sdk/?tab=model#tracing-spans
+[3]: /llm_observability/setup/auto_instrumentation/
+[4]: /llm_observability/setup/?tab=decorators#instrument-your-llm-application
+[5]: /sensitive_data_scanner/
+[6]: /llm_observability/submit_evaluations
```
Comment (alai97) [inline content/en/llm_observability/terms/_index.md:244]:
```suggestion
[6]: /llm_observability/submit_evaluations
[7]: https://github.com/DataDog/llm-observability/blob/main/1-llm-span.ipynb
[8]: https://github.com/DataDog/llm-observability/blob/main/2-workflow-span.ipynb
[9]: https://github.com/DataDog/llm-observability/blob/main/3-agent-span.ipynb
[10]: https://github.com/DataDog/llm-observability/blob/main/4-custom-evaluations.ipynb
```

File: content/en/llm_observability/terms/_index.md [hunk 6]
```diff
@@ -0,0 +1,244 @@
...
+
+### LLM Workflow Monitoring
+
+A workflow trace is composed of a root workflow span with nested LLM, task, tool, embedding, and retrieval spans.
+
+{{< img src="llm_observability/llm-observability-workflow-trace.png" alt="A trace visualizing a more complex LLM workflow" style="width:100%;" >}}
+
+Most LLM applications include operations that surround LLM calls and play a large role in your overall application performance - for example, tool calls to external APIs or preprocessing task steps.
+
+By tracing LLM calls and contextual task or tool operations together under workflow spans, you can unlock more granular insights and a more holistic view of your LLM application.
```
Comment (alai97) [inline content/en/llm_observability/terms/_index.md:137]:
```suggestion
By tracing LLM calls and contextual task or tool operations together under workflow spans, you can unlock more granular insights and a more holistic view of your LLM application.

For a detailed example, see the [LLM Monitoring Jupyter notebook][8] which demonstrates how to create and trace an LLM-powered agent that calls tools and makes decisions based on the data.
```

File: content/en/llm_observability/terms/_index.md [hunk 7]
```diff
@@ -0,0 +1,244 @@
...
+
+### LLM Agent Monitoring
+
+An agent monitoring trace is composed of a root agent span with nested LLM, task, tool, embedding, retrieval, and workflow spans.
+
+{{< img src="llm_observability/llm-observability-agent-trace.png" alt="A trace visualizing an LLM agent" style="width:100%;" >}}
+
+If your LLM application has complex autonomous logic, such as decision-making that can't be captured by a static workflow, you are likely using an LLM Agent. Agents may execute multiple different workflows depending on the user input.
+
+You can instrument your LLM application to trace and group together all workflows and contextual operations run by a single LLM agent as an agent trace.
```
Comment (alai97) [inline content/en/llm_observability/terms/_index.md:147]:
```suggestion
You can instrument your LLM application to trace and group together all workflows and contextual operations run by a single LLM agent as an agent trace.

For a detailed example, see the [LLM Monitoring Jupyter notebook][9] which demonstrates how to create and trace an LLM-powered agent that calls tools and makes decisions based on the data.
```

File: content/en/llm_observability/quickstart.md [hunk 8]
```diff
@@ -1,90 +1,91 @@
...
+This guide uses the [LLM Observability SDK for Python][1]. If your application is written in another language, you can create traces by calling the [API][8] instead.
 
-To run examples from Jupyter notebooks, see the [LLM Observability Jupyter Notebooks repository][12].
+## Setup
 
-## Command-line quickstart
+### Jupyter notebooks
 
-Use the steps below to run a simple Python script that generates an LLM Observability trace.
+To better understand  LLM Observability terms and concepts, you can explore the examples in the [LLM Observability Jupyter Notebooks repository][12]. These notebooks provide a hands-on experience, and allow you to apply these concepts in real time.
```
Comment (alai97) [inline content/en/llm_observability/quickstart.md:22]:
```suggestion
To better understand LLM Observability terms and concepts, you can explore the examples in the [LLM Observability Jupyter Notebooks repository][12]. These notebooks provide a hands-on experience, and allow you to apply these concepts in real time. 
```

**Review by Yun-Kim — APPROVED** [review]
Thank you for the changes @alai97!

**Review by jhgilbert — APPROVED** [review]
This looks great!

File: layouts/partials/nav/left-nav.html [hunk 9]
```diff
@@ -7,7 +7,7 @@
 {{ $ctx := . }}
 {{ $menu := .Site.Menus.main }}
 {{/* Menu identifiers to apply class to */}}
-{{ $excludeAsyc := (slice "api" "dev_tools_metrics_api" "dev_tools_events_api" "integrations_top_level" "tracing api" "watchdog_top_level" "glossary_top_level" "video_top_level" "help_top_level" "other_integrations" "observability_pipelines_reference" "observability_pipelines_reference_sources" "observability_pipelines_reference_transforms" "observability_pipelines_reference_sinks") }}
+{{ $excludeAsyc := (slice "api" "dev_tools_metrics_api" "dev_tools_events_api" "integrations_top_level" "tracing api" "watchdog_top_level" "glossary_top_level" "video_top_level" "help_top_level" "other_integrations" "observability_pipelines_reference" "observability_pipelines_reference_sources" "observability_pipelines_reference_transforms" "observability_pipelines_reference_sinks" "llm_obs_setup_sdk") }}
```
Comment (alai97) [inline layouts/partials/nav/left-nav.html:10]:
```suggestion
{{ $excludeAsyc := (slice "api" "dev_tools_metrics_api" "dev_tools_events_api" "integrations_top_level" "tracing api" "watchdog_top_level" "glossary_top_level" "video_top_level" "help_top_level" "other_integrations" "observability_pipelines_reference" "observability_pipelines_reference_sources" "observability_pipelines_reference_transforms" "observability_pipelines_reference_sinks") }}
```

**Review by LisianeTu — APPROVED** [review]

**Pull request merged**